internal/ecsservicediscovery/decoratedtask_test.go

```diff
@@ -403,18 +402,20 @@ func buildWorkloadEC2BridgeDynamicPort(dockerLabel bool, taskDef bool, serviceNa
 }
 
 func Test_ExportMixedSDTarget_EC2_Bridge_DynamicPort(t *testing.T) {
-	testExportMixedSDTarget_EC2_Bridge_DynamicPort(t, ecs.NetworkModeBridge, 2)
+	networkMode := ecs.NetworkModeBridge
+	testExportMixedSDTarget_EC2_Bridge_DynamicPort(t, &networkMode, 2)
 }
 
 func Test_ExportMixedSDTarget_EC2_Bridge_DynamicPort_With_Implicit_NetworkMode(t *testing.T) {
-	testExportMixedSDTarget_EC2_Bridge_DynamicPort(t, "", 2)
+	testExportMixedSDTarget_EC2_Bridge_DynamicPort(t, nil, 2)
```

Comment:
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html
If users could set networkmode to "" in their ECS task definition, then it would be good to keep the test for "", don't you think?

Reply:
Good instinct! I'm happy to add the test back (and, I guess ensure it behaves same as not specifying the network mode), but don't believe ECS will allow you to set `""` as the value. 

I ran a quick test (setting `networkMode: ""`) and received this error:

```
ClientException:  is not a valid network mode. Must be one of [bridge,host,none,awsvpc]
```

What do you think?

Reply:
Ah, okay.
Thanks for trying it anyways.
Approved.